openmetadata-ui/src/main/resources/ui/src/components/common/SelectableList/SelectableList.component.tsx

```diff
@@ -253,7 +257,7 @@ export const SelectableList = ({
                 size="small"
                 type="primary"
                 onClick={handleUpdateClick}>
-                {t('label.update')}
+                {updating ? 'updating...' : t('label.update')}
               </Button>
```

Comment:
can we use a button loader here, and also try to avoid using normal string and use localization keys if required ?